test/common.py

```diff
@@ -1199,13 +1199,15 @@ def _build_and_run(self, filename, expected_output, args=None, output_nicerizer=
                               interleaved_output=interleaved_output)
       js_output = js_output.replace('\r\n', '\n')
       if expected_output:
+        if type(expected_output) not in [list, tuple]:
+          expected_output = [expected_output]
         try:
           if assert_identical:
             self.assertIdentical(expected_output, js_output)
```

Comment:
How can this line work now, since `expected_output` is an array, while `js_output` is a string?

Reply:
Oh yeah, we don't have any test coverage of that case.. I guess I'll convert this to an assert

Reply:
Oh wait, not I was wrong.. this is fine because `assertIdentical` and `assertContains` both that lists *or* strings, just like all these functions.

Reply:
This change is good as it stands I think

Reply:
I see, makes sense.